### PR TITLE
ntp/settings: do not add duplicated ntp server to nodeconfig

### PR DESCRIPTION
**Pull request by Vicente-Cheng**
**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

**Problem:**
We should avoid adding duplicated NTP servers.

**Solution:**
Both frontend and backend need to add the related handle.

**Related Issue:**
Related: https://github.com/harvester/harvester/issues/3133#issuecomment-1595486084

**Test plan:**
TBD

### CODE REVIEW

File: pkg/controller/master/setting/ntp_server.go [hunk 1]
```diff
@@ -42,6 +43,7 @@ func (h *Handler) syncNTPServer(setting *harvesterv1.Setting) error {
 				return err
 			}
 
+			ntpServers := reGenerateNTPServers(ntpSettings, []string{})
```
Comment (bk201) [inline pkg/controller/master/setting/ntp_server.go]: How about just refactoring `util.GetNTPServers`? This also doesn't make `util.GetNTPServers` an unused function. We can also consider moving the call out of the loop so it only run once.

Comment (Vicente-Cheng) [reply pkg/controller/master/setting/ntp_server.go]: Move to utils.
Also, reduce the duplicated re-generate operation when we do not have nodeconfig.

**Review by guangbochen — APPROVED** [review]
LGTM, thanks.

**Review by bk201 — APPROVED** [review]
LGTM!

**Comment by bk201** [comment]
@mergifyio backport v1.2

**Pull request merged**